### DISCUSSION TITLE
Question regarding adding support for XDEFI Wallet

### DISCUSSION THREAD

**Discussion by loxator** [wagmi-dev/references#120] [Connector request]
### What **novel use-case** does the Connector provide?

Hi there,

 I want to integrate our browser extension XDEFI Wallet (https://xdefi.io) in RainbowKit. Following the breadcrumbs, I stumbled here. Just wanted to confirm that what I am planning is correct before beginning. 
 
I am unsure if it should be a connector as we are multichain wallet and have other providers as well with their on SDKs as can be seen [here](https://sdk.xdefi.io/docs/HEAD/index.html). 
 
 However, for EVM support I assume the below steps should suffice:
 
 1. I need to add an `isXDEFI` property to `InjectedProviderFlags` and `getInjectNames.ts`, and update the equivalent test files.
 
 2. Once the above is merged, I can then add it to RainbowKit.
 
 Please let me know if I am missing anything, and thank you for your help!

### Are the Connector's integrations production-ready and generally available?

Unsure about this, we do have a SDK available

### Are you committed to actively mainting the Connector?

- [X] Yes, my team is or I am committed to actively maintaining the Connector.

**Comment by tmm** [comment]
Hi there! You are welcome to add a XDEFI name mapping. Please open a pull request and we can review.
